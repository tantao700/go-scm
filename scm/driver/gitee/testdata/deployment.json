{
    "url": "https://gitee.com/api/v5/repos/octocat/example/deployments/42/statuses/1",
    "id": 1,
    "node_id": "MDE2OkRlcGxveW1lbnRTdGF0dXMx",
    "state": "success",
    "creator": {
        "login": "octocat",
        "id": 1,
        "node_id": "MDQ6VXNlcjE=",
        "avatar_url": "https://github.com/images/error/octocat_happy.gif",
        "gravatar_id": "",
        "url": "https://gitee.com/api/v5/users/octocat",
        "html_url": "https://github.com/octocat",
        "followers_url": "https://gitee.com/api/v5/users/octocat/followers",
        "following_url": "https://gitee.com/api/v5/users/octocat/following{/other_user}",
        "gists_url": "https://gitee.com/api/v5/users/octocat/gists{/gist_id}",
        "starred_url": "https://gitee.com/api/v5/users/octocat/starred{/owner}{/repo}",
        "subscriptions_url": "https://gitee.com/api/v5/users/octocat/subscriptions",
        "organizations_url": "https://gitee.com/api/v5/users/octocat/orgs",
        "repos_url": "https://gitee.com/api/v5/users/octocat/repos",
        "events_url": "https://gitee.com/api/v5/users/octocat/events{/privacy}",
        "received_events_url": "https://gitee.com/api/v5/users/octocat/received_events",
        "type": "User",
        "site_admin": false
    },
    "description": "Deployment finished successfully.",
    "environment": "production",
    "target_url": "https://example.com/deployment/42/output",
    "created_at": "2012-07-20T01:19:13Z",
    "updated_at": "2012-07-20T01:19:13Z",
    "deployment_url": "https://gitee.com/api/v5/repos/octocat/example/deployments/42",
    "repository_url": "https://gitee.com/api/v5/repos/octocat/example",
    "environment_url": "",
    "log_url": "https://example.com/deployment/42/output"
}