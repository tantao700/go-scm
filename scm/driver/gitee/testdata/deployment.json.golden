{
    "Number": 1,
    "State": 3,
    "Environment": "production",
    "EnvironmentURL": "",
    "Desc": "Deployment finished successfully.",
    "Target": "https://example.com/deployment/42/output"
}
